tests/ha/check_after_reboot.pm

```diff
@@ -76,6 +78,19 @@ sub run {
 
     # Wait for resources to be started
     if (is_sles4sap) {
+        if (get_var('HA_CLUSTER_INIT') && check_var('CLUSTER_NAME', 'hana')) {
+            my $instance_id = get_required_var('INSTANCE_ID');
+            my $sid         = get_required_var('INSTANCE_SID');
+            my $sapadm      = $self->set_sap_info($sid, $instance_id);
```

Comment:
Aren't these values set by a previous module (an SAP module) previously during the test? Seems wrong to set these SAP specific values in a HA test module.

If the reason is reusing a OO method from `lib/sles4sap` to get the sapadm user, I think a better approach would be to add a functional method in `lib/sles4sap` to get the sapadm user, and then use that in `lib/sles4sap::set_sap_info()`.

Reply:
I prefer using `lc($sid) . "adm";`